plugin/src/main/java/io/jenkins/plugins/remotingkafka/GlobalKafkaConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package io.jenkins.plugins.remotingkafka;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import jenkins.model.GlobalConfiguration;
+import net.sf.json.JSONObject;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerRequest;
+
+import java.util.Properties;
+
+@Extension
+public class GlobalKafkaConfiguration extends GlobalConfiguration {
+    private String connectionURL;
+    private String consumerGroupID;
+    private Properties producerProps;
```

Comment:
Imho we should have a follow-up task to stop storing properties on the disk. Fine for now

Reply:
Created a follow-up task: https://issues.jenkins-ci.org/browse/JENKINS-51857